llama_index/llms/azure_openai.py

```diff
@@ -63,6 +63,7 @@ def __init__(
         additional_kwargs: Optional[Dict[str, Any]] = None,
         max_retries: int = 3,
         timeout: float = 60.0,
+        reuse_client: bool = False,
```

Comment:
I think default is True elsewhere. Should this also be true?

Reply:
Good catch -- copilot autocomplete strikes again lol